web/app/components/base/prompt-editor/hooks.ts

```diff
@@ -64,6 +64,7 @@ export const useSelectOrDelete: UseSelectOrDeleteHanlder = (nodeKey: string, com
             editor.dispatchCommand(command, undefined)
 
           node.remove()
+          return true
```

Comment:
> `registerCommand's` callback can return true to signal to the other listeners that the command has been handled and propagation will be stopped.

You can see more detail in the [lexical document](https://lexical.dev/docs/concepts/commands#editorregistercommand).